.pre-commit-config.yaml

```diff
@@ -10,3 +10,8 @@ repos:
   rev: v0.3.1
   hooks:
     - id: ruff
+- repo: https://github.com/astral-sh/uv-pre-commit
+  # uv version.
```

Comment:
I'd like to remove this comment as it's unneeded. Just following the same pattern we have in the file.